### ISSUE TITLE
Need a way to flush the Buffer

### ISSUE THREAD

**Issue by dumbPy** [NVIDIA/nvvl#75]
Would it be possible for you to add a way to flush the buffer and reiterate over the loader without having to exhaust it?  Be it an inefficient way, (than loading all frames and calling `pass`).

**Comment by w550515513** [comment]
yes, it's a problem. The time of one iter from 0.0005s to 0.35s, then flush the buffer and get the new cycle.

**Comment by w550515513** [comment]
![img](https://user-images.githubusercontent.com/14899405/56465593-e71b9e00-6432-11e9-9bb4-9a16dbd2e188.png)
